src/mocks/genre-list.ts

```diff
@@ -1,7 +1,16 @@
+import { DropdownOption } from '../components/MovieForm/MovieForm';
+
 export const GENRE_LIST_MOCK = [
```

Comment:
I believe we can simply map labels from GENRE_LIST_OPTIONS constant  since we use the same values like on selects. And then set active genre using value from constant.